pyiron_atomistics/lammps/interactive.py

```diff
@@ -114,23 +114,35 @@ def interactive_cells_setter(self, cell):
             warnings.warn('set_relative() is deprecated as of 2020-02-26. It is not guaranteed from pyiron_atomistics vers. 0.3')
 
         if is_skewed and is_scaled:
+            self._interactive_lib_command(
+                "change_box all triclinic"
+            )
             self._interactive_lib_command(
                 "change_box all x final 0 %f y final 0 %f z final 0 %f \
-                 xy final %f xz final %f yz final %f triclinic remap units box"
+                 xy final %f xz final %f yz final %f remap units box"
                 % (lx, ly, lz, xy, xz, yz)
             )
         elif is_skewed and not is_scaled:
+            self._interactive_lib_command(
+                "change_box all triclinic"
+            )
             self._interactive_lib_command(
                 "change_box all x final 0 %f y final 0 %f z final 0 %f \
-                xy final %f xz final %f yz final %f triclinic units box"
+                xy final %f xz final %f yz final %f units box"
                 % (lx, ly, lz, xy, xz, yz)
             )
         elif not is_skewed and is_scaled:
+            self._interactive_lib_command(
+                "change_box all ortho"
+            )
             self._interactive_lib_command(
                 "change_box all x final 0 %f y final 0 %f z final 0 %f remap units box"
                 % (lx, ly, lz)
             )
         else:  # is neither skewed nor scaled
+            self._interactive_lib_command(
+                "change_box all ortho"
```

Comment:
@samwaseda Is this what you meant?

Reply:
No this won't work either, because `xy` will still retain the previous value.

Reply:
@samwaseda Then I am not sure what you want and prefer to merge this pull request and address your request in a separate pull request.

Reply:
Wait "I don't understand what you're saying so I'm gonna merge it" is anyway not quite acceptable. Let me explain the problem one more time and then we're gonna talk about whether to still ignore the problem and merge it or not, ok?

So what pyiron does is set `lx ly lz xy xz yz` if the box is skewed, and `lx ly z` if the box is orthogonal. Imagine we have an orthogonal box at the beginning and go through the following case:

- Set `xy=0.1` -> run
- Set `xy=0` -> run

What happens inside pyiron is:

- `xy=0.1`: box is skewed -> `change_box lx ly lz xy xz yz` -> run
- `xy=0`: box is orthogonal -> change_box lx ly lz` -> run

And as you can see in the second step, `xy` remains untouched, while pyiron should have actively set `xy=0`.

I hope this explanation was clear.

Reply:
I understand the physical implications - I just say it was not working before and I do not know how to fix it, because we would have to read the current box inside lammps and then make a decision how to change this box which most likely leads to performance penalties. Therefore I suggest to merge the current pull request because it at least fixes the old elastic constant protocol developed by yury, which is not working with the current master branch.

Reply:
I have the feeling that it's better to leave it then, because, again, going from skewed to orthogonal won't raise an error, but silently returns wrong output. So by making it possible to go from orthogonal to skewed with this PR, we might be making it even more difficult to realise where the problem comes from later.

So I would say, either we solve both problems or nothing. But I'll accept also other decisions if other people prefer a different way. What do you guys think? @liamhuber @pmrv @sudarsan-surendralal

Reply:
Aha. Also from your other comment I understand a bit better. Failing silently is the worst of devils and should be exorcised immediately. I agree entirely that making it work in one direction but not the other makes this an even worse trap.

We definitely don't need the functionality to work both ways, but we should make it fail ASAP, *especially* if we now make it work one way.

Reply:
Ok then maybe the quickest solution is to set `change_box orthogonal`, because then LAMMPS complains, right?

Reply:
> because then LAMMPS complains, right?

*cough* need tests *cough* 😉